validate-policies.py

```
@@ -0,0 +1,43 @@
+for f in sorted(glob.glob('**/*.json', recursive=True)):
+    try:
+        d = json.load(open(f))
+    except Exception as e:
+        print(f'  INVALID JSON  {f}: {e}'); bad += 1; continue
+    if not isinstance(d, dict) or 'Statement' not in d:
+        continue
+    for k in d:
+        if k not in POLICY_KEYS:
+            print(f'  BAD TOP KEY   {f}: {k!r}'); bad += 1
+    stmts = d['Statement']
+    if isinstance(stmts, dict): stmts = [stmts]
```

Ruff (0.16.1)

[error] Multiple statements on one line (colon)

(E701)